std/node/_util/_util_promisify.ts

```diff
@@ -82,21 +81,21 @@ export function promisify(original: Function): Function {
 
   // Names to create an object from in case the callback receives multiple
   // arguments, e.g. ['bytesRead', 'buffer'] for fs.read.
-  // @ts-expect-error TypeScript (as of 3.7) does not support indexing namespaces by symbol
-  const argumentNames = original[kCustomPromisifyArgsSymbol];
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const argumentNames = (original as any)[kCustomPromisifyArgsSymbol];
 
   function fn(...args: unknown[]): Promise<unknown> {
     return new Promise((resolve, reject) => {
-      // @ts-expect-error: 'this' implicitly has type 'any' because it does not have a type annotation
+      // @ts-ignore
```

Comment:
I wasnt able to make the compiler happy with line 90. Ill take any suggestions! 😁

Reply:
```diff
-  function fn(...args: unknown[]): Promise<unknown> {
-    return new Promise((resolve, reject) => {
-      // @ts-ignore
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  function fn(this: any, ...args: unknown[]): Promise<unknown> {
+    return new Promise((resolve, reject) => {
```

Reply:
Does this not block scope `this`  as a parameter? I wouldnt have guessed this as a suggestion

Reply:
Given that `this` is usually implicit, that is

Reply:
> Does this not block scope `this` as a parameter?

Nope, it's a special TS syntax that lets you bind a type to `this` (exactly what we want here). JS wouldn't let you have a parameter called `this`.

Reply:
![mind blown](https://media.giphy.com/media/EldfH1VJdbrwY/giphy.gif)